### PR TITLE
Optimize for build WSA kernels

### PR DESCRIPTION
**Pull request by s1204IT**
`5.15.104.4` will not be released, so commit internal changes first.

### CODE REVIEW

**Comment by s1204IT** [comment]
@tiann
`5.15.104.4` has finally been released, so please approve and review.

**Comment by s1204IT** [comment]
And, I'd like the kernel to be include in `v0.6.9` if possible.

**Review by tiann — CHANGES_REQUESTED** [review]

File: .github/workflows/build-kernel-wsa.yml [hunk 1]
```diff
@@ -91,10 +94,6 @@ jobs:
           declare -A ARCH_MAP=(["x86_64"]="x64" ["arm64"]="arm64")
           cp configs/wsa/config-wsa-${ARCH_MAP[${{ matrix.arch }}]} .config
           make olddefconfig
-          if [ ! -z ${{ vars.EXPECTED_SIZE }} ] && [ ! -z ${{ vars.EXPECTED_HASH }} ]; then
```
[the remark this answers is not in the document]
Comment (s1204IT) [reply .github/workflows/build-kernel-wsa.yml:94]: https://github.com/tiann/KernelSU/pull/1025/commits/acc7530
Reverted

**Review by tiann — APPROVED** [review]

**Comment by s1204IT** [comment]
> And, I'd like the kernel to be include in `v0.6.9` if possible.

@tiann 
Is it possible?

**Comment by tiann** [comment]
> > And, I'd like the kernel to be include in `v0.6.9` if possible.
> 
> @tiann Is it possible?

done

**Pull request merged**